sass/app.scss

```diff
@@ -1,7 +1,103 @@
 /* apply a natural box layout model to all elements, but allowing compnents to change */
+@import "bourbon";
+@import "neat";
+@import "normalize";
+
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
+@import "font-awesome";
+
+/* GLOBAL STYLING */
 html {
   box-sizing: border-box;
 }
 *, *::before, *::after {
   box-sizing: inherit;
 }
+body {
+  background: url(img/bg.jpg);
```

Comment:
This background will be better on the `html` element so that incase the body isn't the full height of the page, it will still be covered.